app/models/legal_aid_application.rb

```diff
@@ -152,16 +149,17 @@ def find_or_create_lead_proceeding_type
   end
 
   def application_proceedings_by_name
-    types = application_proceeding_types.map do |application_proceeding_type|
+    # The name of this method is misleading - as they are no longer sorted by name, but by the order in
```

Comment:
Should we have a ticket to refactor this?

Reply:
I'll do the refactor now.

Reply:
I've just changed the comment  to:
```
    # returns an array of OpenStructs containing:
    # - name of the proceeding type
    # - meaning of the proceeding type
    # - the ApplicationProceedingType
    #
    # in the order they were added to the LegalAidApplication
    # 
```
Probably doesn't need renaming after all.

Reply:
Cool 👍